libs/generic-view/ui/src/lib/interactive/format-message.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React, { useId } from "react"
+import { APIFC } from "generic-view/utils"
+import { FormatMessageConfig, FormatMessageData } from "generic-view/models"
+import {
+  createIntl,
+  createIntlCache,
+  FormattedMessage,
+  RawIntlProvider,
+} from "react-intl"
+import { useSelector } from "react-redux"
+import { selectActiveDeviceConfiguration } from "generic-view/store"
+
+export const FormatMessage: APIFC<FormatMessageData, FormatMessageConfig> = ({
+  data,
+  config,
+  children,
+  ...props
+}) => {
+  const id = useId()
+  const deviceConfig = useSelector(selectActiveDeviceConfiguration)
```

Comment:
⚪ 
Is it necessary to tie this component to the active device state? Should lang come from the API or a default configuration Provider instead?

Reply:
For now only components are generated only in context of active device.